common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -253,7 +254,11 @@ def get_html(self):
                 if val_video_urls["youtube"]:
                     youtube_streams = "1.00:{}".format(val_video_urls["youtube"])
 
-            except edxval_api.ValInternalError:
+                # get video duration
+                video_data = edxval_api.get_video_info(self.edx_video_id.strip())
```

Comment:
Should we confirm if `self.edx_video_id` is not `None`?

Reply:
This code block is under `if self.edx_video_id and edxval_api:` so I don't think we need check None.